conf/merge.go

```diff
@@ -0,0 +1,51 @@
+/*
+ * Copyright (c) 2022, MegaEase
+ * All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package conf
+
+import (
+	"bufio"
+	"bytes"
+	"path/filepath"
+
+	"github.com/mikefarah/yq/v4/pkg/yqlib"
+	log "github.com/sirupsen/logrus"
+)
+
+// mergeYamlFiles merges yaml files using yq(https://github.com/mikefarah/yq)
+func mergeYamlFiles(path string) ([]byte, error) {
+	files, err := filepath.Glob(path + "/*.yaml")
```

Comment:
for `path + "/*.yaml"`, it's better to use `filepath.Join()` this function can help to deal with the issue of cross-platform - Windows uses `\`, linux uses `/` for path separator.